dodola/core.py

```diff
@@ -231,3 +232,26 @@ def xclim_remove_leapdays(ds):
     """
     ds_noleap = convert_calendar(ds, target="noleap")
     return ds_noleap
+
+
+def apply_wet_day_frequency_correction(ds, process):
+    """
+
+    Parameters
+    ----------
+    ds : xr.Dataset
+    process : str
```

Comment:
Document the choices available:

```suggestion
    process : {"pre-process", "post-process"}
```